### ISSUE TITLE
License information

### ISSUE THREAD

**Issue by jelmd** [ramyadirsumilli/rest_owl_api#21]
Since lib/*.jar are third-party products, there should be a lib/readme.txt or NOTICE.txt etc., which lists the origin URLs for it as well as the type of the License or where to obtain it (URL).

**Comment by ramyadirsumilli** [comment]
I have updated the information. 
Not all the .jar files are required. Now `lib/` is only contains required jar files.

**Comment by jelmd** [comment]
Hmmm, ok, but lib/java-json.jar is IMHO not acceptable, since the given URL doesn't provide any information about its writer(s) (possible copyright holders) nor documentation about it. Furthermore [JSR 353](https://jcp.org/en/jsr/detail?id=353) got finally approved ~ 3 years ago and thus will probably show up in the next JRE/J2EE releases ... The download page on the mentioned link also contains a link to the reference implementation of the "Java API for JSON Processing"...

BTW: Do we still need the lib/README.md ?

**Comment by ramyadirsumilli** [comment]
updated. 
please check if it is correct ?

**Comment by jelmd** [comment]
Nope. You should not betray the user! Saying that lib XY from Z is used, but actually using something different (in this case still the obscure version from the dark), is a No-Go!

So please replace java-json.jar with the Reference-Implementation mentioned above, and correct the link as well (currently the link refers to the API documentation, but not the place, where one may obtain it).
